sample_project/Assets/SampleViewer/Resources/InputActions/InputActions.cs

```diff
@@ -2,7 +2,7 @@
 // <auto-generated>
 //     This code was auto-generated by com.unity.inputsystem:InputActionCodeGenerator
 //     version 1.7.0
-//     from Assets/SampleViewer/Samples/Geometry/InputActions.inputactions
```

Comment:
Do we need this change to the InputActions.cs file? I believe this is an auto generated change when using a newer version of Unity, like Unity 6.

Reply:
Dont need it. Must have been added by mistake.